.changeset/cool-carrots-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/ui-react': patch
+---
```

Comment:
Since we didn't do the libraries release, changeset were not removed automatically. Do we want to remove them manually?

Reply:
Maybe! Does it have an impact on the release?

Reply:
It's better to leave them, it's when there are none that a release can trigger.

Reply:
We do not need to remove them for docs-release, because docs release is entirely done by Amplify Hosting.